server/src/main/java/org/opensearch/index/shard/IndexingStats.java

```diff
@@ -226,6 +230,8 @@ public void add(Stats stats) {
             if (getDocStatusStats() != null) {
                 getDocStatusStats().add(stats.getDocStatusStats());
             }
+            // Aggregate maxLastIndexRequestTimestamp using Math.max
```

Comment:
This comment isn't necessary. Reading the code will tell you the same thing. Good comments usually provide some information or context that isn't obvious from reading the code.